### PR TITLE
- Removed some old never to be used code

### PR DESCRIPTION
**Pull request by sjoerdbr**
+ added navbar
+ changed route to make module overview default page

### CODE REVIEW

File: src/app/app.routing.ts [hunk 1]
```diff
@@ -1,17 +1,22 @@
 import {NgModule} from '@angular/core';
 import {RouterModule, Routes} from '@angular/router';
 import {ErrorComponent} from '../util/error/error.component';
-import {ModuleComponent} from "./module-component/module.component";
-import {SkillMatrixComponent} from "./skillmatrix-component/skillmatrix.component";
+import {ModuleComponent} from "./components/module-component/module.component";
```
Comment (NNieuwenhuis) [inline src/app/app.routing.ts]: single quote

File: src/app/app.routing.ts [hunk 2]
```diff
@@ -1,17 +1,22 @@
...
+import {SkillMatrixComponent} from "./components/skillmatrix-component/skillmatrix.component";
```
Comment (NNieuwenhuis) [inline src/app/app.routing.ts]: single quote

File: src/app/app.routing.ts [hunk 3]
```diff
@@ -1,17 +1,22 @@
...
+import {ModuleOverviewComponent} from "./components/module-overview-component/module-overview.component";
```
Comment (NNieuwenhuis) [inline src/app/app.routing.ts]: single quote

Comment (NNieuwenhuis) [inline src/app/app.routing.ts]: Doesn't work after selecting a curriculum nothing happens!

File: src/app/app.routing.ts [hunk 4]
```diff
@@ -1,17 +1,22 @@
...
   {
+    path: 'moduleoverview',
+    component: ModuleOverviewComponent
+  },
+  {
     path: 'module',
     component: ModuleComponent
   },
   {
     path: '',
```
Comment (NNieuwenhuis) [inline src/app/app.routing.ts]: Doesn't work after selcting a curriculum nothing happens!

Comment (NNieuwenhuis) [inline src/app/app.routing.ts:17]: Doesn't work after a curriculum is selected nothing happens!

File: src/util/nav/nav.html [hunk 5]
```diff
@@ -3,19 +3,12 @@
...
-      <li class="nav-item active">
-        <a class="nav-link" href="#">Home <span class="sr-only">(current)</span></a>
-      </li>
       <li class="nav-item">
-        <a class="nav-link" href="#">Link</a>
+        <a class="nav-link" href="moduleoverview">Home</a>
       </li>
       <li class="nav-item">
-        <a class="nav-link disabled" href="#">Disabled</a>
+        <a class="nav-link" href="module">Mock module(will be removed)</a>
```
Comment (NNieuwenhuis) [inline src/util/nav/nav.html:10]: Why is i in here, if its never used?

**Review by NNieuwenhuis — CHANGES_REQUESTED** [review]
There are some errors.

File: src/util/nav/nav.html (hunk 5 above)
Comment (sjoerdbr) [reply src/util/nav/nav.html:10]: For development speed

File: src/app/app.routing.ts (hunk 4 above)
Comment (sjoerdbr) [reply src/app/app.routing.ts:17]: I didn't change anything on the moduleoverview component part. This doesn't contribute to it not working.

**Review by NNieuwenhuis — APPROVED** [review]

**Pull request merged**